lms/djangoapps/mobile_api/models.py

```diff
@@ -76,3 +81,28 @@ def save(self, *args, **kwargs):
         """ parses version into major, minor and patch versions before saving """
         self.major_version, self.minor_version, self.patch_version = utils.parsed_version(self.version)
         super(AppVersionConfig, self).save(*args, **kwargs)
+
+
+class IgnoreMobileAvailableFlagConfig(ConfigurationModel):  # pylint: disable=W5101
+    """
+    Configuration for the mobile_available flag.
+
+    Setting mobile_available_override to true will always grant access in
+    utils.mobile_course_access and will ignore the mobile_available flag
+    when returning the list of enrolled courses in
+    users.views.UserCourseEnrollmentsList
+    """
+    mobile_available_override = models.BooleanField(
```

Comment:
There's already an `enabled` field that comes standard with the `ConfigurationModel`.  Can we use that instead?